java/clients/src/main/java/sleeper/clients/deploy/DeployNewInstance.java

```diff
@@ -113,11 +113,12 @@ public void deploy() throws IOException, InterruptedException {
         LOGGER.info("jarsDirectory: {}", jarsDirectory);
         LOGGER.info("sleeperVersion: {}", sleeperVersion);
 
-
+        Properties tagsProperties = loadProperties(templatesDirectory.resolve("tags.template"));
+        tagsProperties.setProperty("Name", instanceId);
```

Comment:
It would be nice to use "InstanceId" instead of "Name". We can also change the name of that field in the template.